pkg/datasources/resource_monitors.go

```diff
@@ -49,8 +49,10 @@ func ResourceMonitors() *schema.Resource {
 
 func ReadResourceMonitors(d *schema.ResourceData, meta interface{}) error {
 	db := meta.(*sql.DB)
+	client := sdk.NewClientFromDB(db)
+	ctx := context.Background()
 
-	account, err := snowflake.ReadCurrentAccount(db)
+	account, err := client.ContextFunctions.Current(ctx)
```

Comment:
i think this function should be named CurrentAccount()  https://docs.snowflake.com/en/sql-reference/functions/current_account. Also should this be put in a  helper struct so it can be something like client.Context.Sessions.CurrentAccount()? Or maybe just client.ContextSessions.CurrentAccount?

Reply:
We already have current account check another comment).

I don't understand the second part, though. It's already exposed in client's ContextFunctions interface.

Reply:
Renamed to CurrentSessionDetails (because CurrentSession was already taken too).